packages/apps/src/SideBar/SideBarScroll.tsx

```diff
@@ -7,6 +7,7 @@ const SideBarScroll = styled.div.attrs({
   @media (min-width:1000px){
     .expanded & {
       position: fixed;
+      width: 12%;
```

Comment:
Updated the css property to fix the issue 
![Uploading Screen Shot 2020-10-28 at 12.41.18 PM.png…]()